Gopkg.toml

```diff
@@ -29,17 +29,17 @@
   branch = "master"
   name = "github.com/golang/glog"
 
-[[constraint]]
-  name = "github.com/onsi/gomega"
-  version = "1.4.0"
-
 [[constraint]]
   name = "gopkg.in/yaml.v2"
   version = "2.2.1"
 
 [[constraint]]
-  branch = "master"
   name = "k8s.io/api"
+  version = "kubernetes-1.9.0"
```

Comment:
Why are we writing version as kubernetes-1.9.0 and not as "1.9.0"

Reply:
Kubernetes releases don't use short syntax, they follow `kubernetes-version`.

Reply:
Ok ! Thanks !